### PR TITLE
Remove unused code in mrtarget/common/LookupHelpers.py

### PR DESCRIPTION
**Pull request by peatroot**
This PR:
* used vulture to look for dead code
* searched manually to confirm lack of usage
* deleted dead code

### CODE REVIEW

File: mrtarget/common/LookupHelpers.py [hunk 1]
```diff
@@ -139,7 +137,6 @@ def _get_hpo(self):
         obj = OntologyClassReader()
         hpo_uri = Config.ONTOLOGY_CONFIG.get('uris', 'hpo')
         opentargets_ontologyutils.hpo.get_hpo(obj, hpo_uri)
-        obj.rdf_graph = None
```
Comment (afaulconbridge) [inline mrtarget/common/LookupHelpers.py]: I think this line is important in saving memory by free objects that should no longer be used for garbage collection.

File: mrtarget/common/LookupHelpers.py [hunk 2]
```diff
@@ -150,7 +147,6 @@ def _get_mp(self):
         obj = OntologyClassReader()
         mp_uri = Config.ONTOLOGY_CONFIG.get('uris', 'mp')
         opentargets_ontologyutils.mp.load_mammalian_phenotype_ontology(obj, mp_uri)
-        obj.rdf_graph = None
```
Comment (afaulconbridge) [inline mrtarget/common/LookupHelpers.py]: Also here - I think this line is important in saving memory by free objects that should no longer be used for garbage collection.

File: mrtarget/common/LookupHelpers.py (hunk 1 above)
Comment (peatroot) [reply mrtarget/common/LookupHelpers.py]: Done!

File: mrtarget/common/LookupHelpers.py (hunk 2 above)
Comment (peatroot) [reply mrtarget/common/LookupHelpers.py]: Done!

**Pull request merged**